modules/persistent-storage-using-a-shared-configmap-in-a-pod.adoc

```diff
@@ -0,0 +1,51 @@
+:_content-type: PROCEDURE
+
+[id="persistent-storage-using-a-shared-configmap-in-a-pod_{context}"]
+= Using a shared configMap in a pod
+
+[role="_abstract"]
+
+.Prerequisites
+
+* You created a `SharedConfigMap` object for the config map that you want to share across namespaces in the cluster.
+* You have privileges to perform the following actions:
+  * Create pods and access instances of available `sharedconfigmaps.sharedresource.openshift.io`. In other words, you can run `oc get sharedconfigmaps` and get a non-empty list back.
+  * Determine if any of the service accounts available to you in your namespace are allowed to use a given `SharedConfigMap` resource. In other words, you can run `oc adm policy who-can use <identifier of specific SharedConfigMap>` to see if a service account in your namespace is listed.
+  * Establish, or ask someone to establish, the necessary role-based access control (RBAC) so that you can discover `SharedConfigMaps` resources and allow service accounts to use `SharedConfigMaps` resources.
```

Comment:
```suggestion
  * If either of the prior two steps could not satisfy your needs, establish, or ask someone to establish, the necessary role-based access control (RBAC) so that you can discover `SharedConfigMaps` resources and allow service accounts to use `SharedConfigMaps` resources.
```

Reply:
Updated here and similarly in modules/persistent-storage-using-a-sharedconfigmap-object-in-a-pod.adoc.
Thanks.